### ISSUE TITLE
Standardize a column

### ISSUE THREAD

**Issue by reza1615** [man-group/dtale#271]
I would be helpful to Standardize a column. it can be inplace or new column
https://scikit-learn.org/stable/auto_examples/preprocessing/plot_all_scaling.html#sphx-glr-auto-examples-preprocessing-plot-all-scaling-py
In my opinion these algorithms are useful
* handles outliers:
  * RobustScaler
  * QuantileTransformer 
  * PowerTransformer
* Suffers from outliers
  * Normalizer
  * StandardScaler
  * MinMaxScaler
  * MaxAbsScaler

**Comment by aschonfeld** [comment]
@reza1615 I'm trying to start building out this code and it seems a little hacky.  Is this the best way?
```
import pandas as pd
from sklearn.preprocessing import PowerTransformer
from numpy.random import randn

df = pd.DataFrame(dict(a=randn(1000)))
power = PowerTransformer(method='yeo-johnson', standardize=True)
df.loc[:, 'a_standardized'] = power.fit_transform(df.a.values.reshape((len(df.a),1))).reshape(-1)
```

**Comment by reza1615** [comment]
Here is my snippet  code. As you see if the list in front of scalers is empty it doesn't do any action
``` python
import pandas as pd

from sklearn.compose import ColumnTransformer
from sklearn.base import BaseEstimator, TransformerMixin

from sklearn.preprocessing import MinMaxScaler
from sklearn.preprocessing import MaxAbsScaler
from sklearn.preprocessing import StandardScaler
from sklearn.preprocessing import RobustScaler
from sklearn.preprocessing import Normalizer
from sklearn.preprocessing import QuantileTransformer
from sklearn.preprocessing import PowerTransformer

class NoTransformer(BaseEstimator, TransformerMixin):
    def fit(self, X, y=None):
        return self

    def transform(self, X):
        assert isinstance(X, pd.DataFrame)
        return X

def apply_ColumnTransformer(df):
    column_trans = ColumnTransformer(
        ([
            ('', NoTransformer(), list(df.columns)),
            ('std', StandardScaler(), ['a']),
            ('MinMaxScaler', MinMaxScaler(), []),
            ('MaxAbsScaler', MaxAbsScaler(), []),
            ('StandardScaler', StandardScaler(), []),
            ('RobustScaler', RobustScaler(), []),
            ('Normalizer', Normalizer(), []),
            ('QuantileTransformer', QuantileTransformer(), []),
            ('yeo-johnson', PowerTransformer(method='yeo-johnson', standardize=True), ['a', 'b']),
        ]), remainder="passthrough")

    transformed = column_trans.fit_transform(df)
    return transformed,column_trans

def colum_names(column_trans):
    new_col_names = []
    for i in range(len(column_trans.transformers)):
        col_name=column_trans.transformers[i][0]
        new_col_names +=[col_name + '_' + s if col_name else s for s in column_trans.transformers[i][2]]
    return new_col_names

## create df
from numpy.random import randn
df = pd.DataFrame(dict(a=randn(1000), b=randn(1000)))

transformed,column_trans=apply_ColumnTransformer(df)
new_col_names= colum_names(column_trans)
df=pd.DataFrame(transformed, columns=new_col_names)

```
**Result**
![image](https://user-images.githubusercontent.com/2202483/96117741-88fd3900-0eb8-11eb-9215-cfcd50e86b3e.png)

**Comment by aschonfeld** [comment]
This is great, thank you

**Comment by aschonfeld** [comment]
included in v1.18.0
